export class RecursoNaoEncontradoError extends Error {
  constructor() {
    super("Recurso n√£o encontrado.");
  }
}
